functions/go/source-gcloud-config/kptfile/v1/types.go

```diff
@@ -0,0 +1,328 @@
+// Copyright 2021 Google LLC
```

Comment:
can this be obtained by import from kpt repository rather than copying the file?

Reply:
I swear I asked the similar questions before. My understanding is that kpt does not want function developers to vendor kpt repo. FYI, the kptfile schema is in [kpt-function-sdk  repo](https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/go/pkg/api/kptfile/v1/types.go) and the kpt repo keeps a copy of it.

Reply:
Thank you for the added context! vendoring from kpt-function-sdk seems ideal.

@loudej fyi re: authoritative copy of types.go relevant to OCI work

Reply:
done. Actually I changed the config so the Kptfile is not needed.